source/application/IDEAS/IDEASAppController.cpp

```diff
@@ -206,6 +208,9 @@ void IDEASAppController::onEnergyConnected(bool connected){
 
 		connect(xasScanConfiguration, SIGNAL(totalTimeChanged(double)), xasScanConfigurationHolder3_, SLOT(updateOverallScanTime(double)));
 		xasScanConfigurationHolder3_->updateOverallScanTime(xasScanConfiguration->totalTime());
+
+		connect(ideasKETEKDetailedDetectorViewWithSave_, SIGNAL(peakingChanged(const QString&)), xasScanConfigurationView_, SLOT(onDetectorSettingsChange(const QString&)));
+		connect(ideas13ElementGeDetailedDetectorViewWithSave_, SIGNAL(peakingChanged(const QString&)), xasScanConfigurationView_, SLOT(onDetectorSettingsChange(const QString&)));
```

Comment:
@iainworkman The connection is made here, in the app controller. Is that what you meant? Would there be a better place for it?